client/app/actions/message_action_creator.coffee

```diff
@@ -1,7 +1,7 @@
 {ActionTypes}   = require '../constants/app_constants'
 XHRUtils        = require '../libs/xhr'
 
-MessageGetters  = require '../puregetters/message'
+MessageGetters  = require '../puregetters/messages'
```

Comment:
Mmmh, to be honnest, my tests so far are more of the randomly click everywhere and fix the errors spawning in browser console. I will try to make a list